src/index.html

```diff
@@ -5,8 +5,323 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="attr-req-value"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="body">
+    <header class="header">
+      <a class="logo header__elem" href="#home"><img src="./images/logo.png" alt="logo"></a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="https://www.moyo.ua/ua/portal_brand/apple/">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="https://www.moyo.ua/ua/search/new/?q=sumsung">Samsung</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="https://www.moyo.ua/ua/telecommunication/smart/">Smartphones</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link"
+              data-qa="nav-hover"
+              href="https://www.moyo.ua/ua/comp-and-periphery/"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="https://www.moyo.ua/ua/gadgets/">Gadgets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="https://www.moyo.ua/ua/tablet_el_knigi/">Tablets</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="https://www.moyo.ua/ua/foto_video/photo_video/">Photo</a>
+          </li>
+          <li class="nav__item">
+            <a class="nav__link" href="https://www.moyo.ua/ua/foto_video/video_tech/">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
```

Comment:
```suggestion
      </nav>
    </header>
    
    <main class="main">
```
blank line between multiline components